woocommerce/woocommerce-gutenberg-products-block
# Convert Textarea component to TS

Converts the Textarea component used in the checkout order notes to TS.

### Testing

#### Automated Tests

### Manual Testing

How to test the changes in this Pull Request:

1. Add the "Checkout block" to a page
2. Add a product to the cart and go to the checkout page with the block
3. Check the "Add a note to your order" checkbox
4. Check that the textarea for adding notes is showing and can be used.
